### PR TITLE
Add more coverage about the course

### PR DESCRIPTION
**Pull request by mgeisler**
There has now been three articles about the course.

### CODE REVIEW

**Comment by mgeisler** [comment]
We can move this to a separate document later if the list grows :smile:

File: README.md [hunk 1]
```diff
@@ -30,8 +30,14 @@ trigger when going through the code samples. We hope to improve on this via
 
 ## Press
 
-- [Scaling Rust Adoption Through Training](https://security.googleblog.com/2023/09/scaling-rust-adoption-through-training.html):
+Articles and blog posts from around the web which cover Comprehensive Rust:
+
+- 2023-09-08: _[Teaching Rust in 5 days](https://mo8it.com/blog/teaching-rust/)_.
```
Comment (mgeisler) [inline README.md:35]:
```suggestion
- 2023-09-08:
  _[Teaching Rust in 5 days](https://mo8it.com/blog/teaching-rust/)_.
```

File: README.md [hunk 2]
```diff
@@ -30,8 +30,14 @@ trigger when going through the code samples. We hope to improve on this via
...
+  Comprehensive Rust was used as a base for a 5-day university class on Rust.
+- 2023-09-21: _[Scaling Rust Adoption Through Training](https://security.googleblog.com/2023/09/scaling-rust-adoption-through-training.html)_.
```
Comment (mgeisler) [inline README.md:37]:
```suggestion
- 2023-09-21:
  _[Scaling Rust Adoption Through Training](https://security.googleblog.com/2023/09/scaling-rust-adoption-through-training.html)_.
```

File: README.md [hunk 3]
```diff
@@ -30,8 +30,14 @@ trigger when going through the code samples. We hope to improve on this via
...
   We published a blog post with details on the development of the course.
+- 2023-10-02: _[In Search of Rust Developers, Companies Turn to In-House Training](https://www.darkreading.com/application-security/google-microsoft-take-refuge-in-rust-languages-better-security)_.
```
Comment (mgeisler) [inline README.md:39]:
```suggestion
- 2023-10-02:
  _[In Search of Rust Developers, Companies Turn to In-House Training](https://www.darkreading.com/application-security/google-microsoft-take-refuge-in-rust-languages-better-security)_.
```

**Review by henrif75 — APPROVED** [review]

**Pull request merged**